purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -144,6 +144,12 @@ class Purchases internal constructor(
     val isAnonymous: Boolean
         get() = identityManager.currentUserIsAnonymous()
 
+    /**
+     * The currently configured store
+     */
+    val store: Store
+        get() = appConfig.store
```

Comment:
Could we add a very simple test that makes sure this is the expected value?

Reply:
Done!